dev/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# About Image: Use LTS version for maintenance purpose and use alpine to reduce image size.
+FROM node:10-alpine
+
+LABEL vendor="Dynamic Lab"
+LABEL maintainer="pangxiaoxi@gmail.com, cndream@gmail.com"
+LABEL description="dv-web dev docker image"
+
+ARG DEV_HOME
+
+WORKDIR $DEV_HOME
+
+# run npm install at CMD level, so that we don't need to build new images again and again
+CMD cd app && \
+    npm install --quiet &&\
+    npm start debug
```

Comment:
If we try to use npm to start the react app, then here should be `npm run start` so that npm will check the package.json file to run the start script

Reply:
Are they different? Based on http://thecodebarbarian.com/3-neat-tricks-with-npm-run, `npm start` is short for `npm run start`